tests/queries/0_stateless/03172_error_log_table_not_empty.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+# Get the previous number of errors for 111, 222 and 333
+errors_111=$($CLICKHOUSE_CLIENT -q "SELECT sum(value) FROM system.error_log WHERE code = 111")
+errors_222=$($CLICKHOUSE_CLIENT -q "SELECT sum(value) FROM system.error_log WHERE code = 222")
+errors_333=$($CLICKHOUSE_CLIENT -q "SELECT sum(value) FROM system.error_log WHERE code = 333")
+
+# Throw three random errors: 111, 222 and 333 and wait for more than collect_interval_milliseconds to ensure system.error_log is flushed
+$CLICKHOUSE_CLIENT -mn -q "
+SELECT throwIf(true, 'error_log', toInt16(111)) SETTINGS allow_custom_error_code_in_throwif=1; -- { serverError 111 }
+SELECT throwIf(true, 'error_log', toInt16(222)) SETTINGS allow_custom_error_code_in_throwif=1; -- { serverError 222 }
+SELECT throwIf(true, 'error_log', toInt16(333)) SETTINGS allow_custom_error_code_in_throwif=1; -- { serverError 333 }
+SELECT sleep(2) format NULL;
```

Comment:
It should be marked as `Tags: no-fasttest` since it uses timeouts.

Reply:
the second type of error is stale, it from the old version of the test ` AND event_time > now() - INTERVAL 1 MINUTE;`

Reply:
Systems tables are created lazily. It could be not created yet when we read from it.

Reply:
fixing it https://github.com/ClickHouse/ClickHouse/pull/66093

Reply:
>It should be marked as Tags: no-fasttest since it uses timeouts.

[Here's](https://github.com/ClickHouse/ClickHouse/pull/65725#discussion_r1654928104) the reasoning why I closed the PR that added precisely that tag

>the second type of error is stale, it from the old version of the test  AND event_time > now() - INTERVAL 1 MINUTE;

That was part of the original test. The timing issue was fixed precisely in this PR. Are you sure the error you see is not from a prior and flaky version of the test?

>Systems tables are created lazily. It could be not created yet when we read from it.

I don't follow you. At what point is the test reading from the table without flushing logs first?

Reply:
> > It should be marked as Tags: no-fasttest since it uses timeouts.
> 
> [Here's](https://github.com/ClickHouse/ClickHouse/pull/65725#discussion_r1654928104) the reasoning why I closed the PR that added precisely that tag

No. You were trying to add `Tags: no-parallel`. I'm talking about `Tags: no-fasttest` because the test uses timeouts.

> > the second type of error is stale, it from the old version of the test  AND event_time > now() - INTERVAL 1 MINUTE;
> 
> That was part of the original test. The timing issue was fixed precisely in this PR. Are you sure the error you see is not from a prior and flaky version of the test?

That error from the test before you changes. Your changes fixed that error, that is good.

> > Systems tables are created lazily. It could be not created yet when we read from it.
> 
> I don't follow you. At what point is the test reading from the table without flushing logs first?

here
`errors_111=$($CLICKHOUSE_CLIENT -q "SELECT sum(value) FROM system.error_log WHERE code = 111")`

Reply:
>No. You were trying to add Tags: no-parallel. I'm talking about Tags: no-fasttest because the test uses timeouts.

Right, but since it's not using timeouts anymore it shouldn't be needed, right?

>here
errors_111=$($CLICKHOUSE_CLIENT -q "SELECT sum(value) FROM system.error_log WHERE code = 111")

You're absolutely right, my bad

Reply:
> Right, but since it's not using timeouts anymore it shouldn't be needed, right?

Yes. This is why I fight for it and trying remove timeouts. It is better to leave the test to work as fast-test. Then mark it as `no-fasttest`.

Reply:
I see. The test takes ~6s to complete. If that's too long for a fast-test (I don't know where the limit is) let's add the `no-fasttest` :+1: